cellbase-lib/src/main/java/org/opencb/cellbase/lib/builders/clinical/variant/ClinVarIndexer.java

```diff
@@ -266,13 +266,34 @@ private boolean updateRocksDB(AlleleLocationData alleleLocationData, PublicSetTy
                 }
 
                 // parse RCVs
-                String accession = publicSet.getReferenceClinVarAssertion().getClinVarAccession().getAcc();
-                String clinicalSignficanceDescription = publicSet.getReferenceClinVarAssertion()
-                        .getClinicalSignificance()
-                        .getDescription();
-                String reviewStatusName = publicSet.getReferenceClinVarAssertion().getClinicalSignificance()
-                        .getReviewStatus().name();
-                List<ObservationSet> getObservedIn = publicSet.getReferenceClinVarAssertion().getObservedIn();
+                String accession = null;
+                try {
+                    accession = publicSet.getReferenceClinVarAssertion().getClinVarAccession().getAcc();
+                } catch (Exception e) {
+                    logger.warn("Error getting accession\n" + StringUtils.join(e.getStackTrace(), "\n"));
```

Comment:
Better let the logger to print the stacktrace:
```
logger.warn("Error getting accession", e);
```

and you can also indicate what will happen with the accession:
```
logger.warn("Error getting accession. Ignore error and leave accession as null.", e);
```
This applies to all other fields read in this method